go.mod

```diff
@@ -36,6 +35,6 @@ require (
 )
 
 replace (
-	github.com/dgrijalva/jwt-go => github.com/form3tech-oss/jwt-go v3.2.1+incompatible
+	github.com/golang-jwt/jwt => github.com/golang-jwt/jwt v3.2.1+incompatible
```

Comment:
I think you can get rid of the replacement here and directly specify `github.com/golang-jwt/jwt v3.2.1+incompatible` in line 17 since all import paths are already adjusted and seem to work.

Reply:
Thanks for the feedback! Regarding this we have deps that use those JWT libs as well, and we're aiming to force the use of github.com/golang-jwt/jwt in those as well.

Reply:
Looks like you're 100% accurate, for some reason fosite's use of the other jwt-go lib doesn't cause this to show up in our go.sum.